src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -105,4 +85,96 @@ private void ClearExistingSequences()
             Destroy(sequencesPanel.transform.GetChild(i).gameObject);
         }
     }
+
+    /**
+     * <summary>
+     * Search a directory for any *.json files, then attempt to read them as Bot Sequences
+     * </summary>
+     * <param name="path">The directory to look for Bot Sequences json files</param>
+     * <returns>List of tuples containing (file name, Bot Sequence) entires</returns>
+     */
+    private IList<(string, BotSequence)> EnumerateSequencesInDirectory(string path)
+    {
+        var sequenceFiles = Directory.EnumerateFiles(path, "*.json");
+        return sequenceFiles
+            .Select(fileName =>
+            {
+                try
+                {
+                    return (Path.GetFileNameWithoutExtension(fileName), BotSequence.LoadSequenceJsonFromPath(fileName));
+                }
+                catch (Exception exception)
+                {
+                    Debug.Log($"Error reading Bot Sequences from {fileName}: {exception}");
+                    return (null, null);
+                }
+            })
+            .Where(s => s.Item2 != null)
+            .ToList();
+    }
+    
+    /**
+     * <summary>
+     * Load Bot Sequences relative to if we're running in the Unity editor, or in a packaged build. For the packaged
+     * build we will first check the persistent data path for any Sequence files, and then check Resources for any. We
+     * will use the persistent data path as an override, so any Sequences that are found in Resources will only be kept
+     * if they are not also within the persistent data path
+     * </summary>
+     * <returns>List of Bot Sequences</returns>
+     */
+    private IList<BotSequence> ResolveSequenceFiles()
+    {
+#if UNITY_EDITOR
+        const string sequencePath = "Assets/RegressionGames/Resources/BotSequences";
+        if (!Directory.Exists(sequencePath))
+        {
+            return new List<BotSequence>();
+        }
+
+        return EnumerateSequencesInDirectory(sequencePath)
+            .Select(s => s.Item2)
+            .ToList();
+#else
+        IList<(string, BotSequence)> sequences = new List<(string, BotSequence)>();
+
+        // 1. check the persistentDataPath for sequences
+        var persistentDataPath = Application.persistentDataPath + "/BotSequences";
+        if (Directory.Exists(persistentDataPath))
+        {
+            sequences = EnumerateSequencesInDirectory(persistentDataPath);
+        }
+    
+        // 2. load Sequences from Resources, while skipping any that have already been fetched from the
+        //    persistentDataPath. We will compare Sequences by their filename (without extension), and by the actual
+        //    Sequence name
+        const string runtimePath = "BotSequences";
+        var jsons = Resources.LoadAll(runtimePath, typeof(TextAsset));
+        foreach (var jsonObject in jsons)
+        {
+            try
+            {
+                var resourceFilename = jsonObject.name;
```

Comment:
@RG-nAmKcAz hopefully this works for comparing loaded Sequences for a build of a game.

I'm grabbing the filename (without the file extension) when reading from persistent data, and excluding any Sequences loaded from Resources if they have the same file name (and Sequence name within the file). From what I read, this `jsonObject.name` call should get a Resources file name without its extension